book/src/remapping.md

```diff
@@ -0,0 +1,38 @@
+# Key Remapping
+
+One-way key remapping is supported via a simple TOML configuration file.
+
+To remap keys, write a `keymap.toml` file in your `helix` configuration
+directory (default `~/.config/helix` in Linux systems) with a structure like
+this:
+
+```toml
+# At most one section each of 'Normal', 'Insert' and 'Select'
+[Normal]
+a = "w" # Maps the 'a' key to 'w' (move to next word)
+w = "i" # Maps the 'w' key to 'i' (enter insert mode)
+C-S-Esc = "f" # Maps Control-Shift-Escape to 'f' (find char)
```

Comment:
I don't think mapping from key to key is a good idea even for a start. Mapping to function maybe.

Reply:
I think both are useful, I just wanted to chat to you guys first about how to approach the command mapping. Maintaining a &static str to Command map is a bit of a maintenance burden (not a big deal though). 

The reason why I think both are useful is that it reduces the context that is needed when you want to make a quick change. Something new to Helix can just say "I want to map `i` to `w`", and they don't need to go somewhere to read how the "move to next word" command is called.